docs/iam-usage.rst

```diff
@@ -0,0 +1,148 @@
+Using IAM For Users
+===================
+
+.. note:: This document assumes basic knowledge of Google Cloud IAM,
+   see `the docs <https://cloud.google.com/iam/docs/>`_ for details
+
+Resources that implement the IAM interface provide the following methods:
+
+Policy Editing Convenience Methods
+----------------------------------
+
+Common keyword arguments:
+
+- ``retries`` number of times to retry in the
+   even of an interrupted transaction, (not available in ``set_policy``)
+- ``version`` version number of the new policy. Not used for transactionality.
+   provided for end-user use. Left blank, will increment the version by 1.
+
+
+>>> resource.add_role(iam.user('alice@example.com'), iam.roles.OWNER.name)
+True
+
+The first argument is one of the IAM "Member" types:
+
+- ``iam.user(email)`` an individual Google account
```

Comment:
It's a method. Members are just very specifically formatted strings: So a "user" is just a string of the form: `'user:alice@example.com'` while a "domain" is just `'domain:example.com'`

Hence my plan was just to provide safe aliases for these: 

```
user = 'user:{}'.format
domain = 'domain:{}'.format
```